server/src/bootstrap/search.ts

```diff
@@ -0,0 +1,27 @@
+import { Client } from '@opensearch-project/opensearch'
+import fs from 'fs'
+import { baseConfig, Environment } from './config/base'
+import { searchConfig } from './config/search'
+
+const searchHost = searchConfig.host
+const protocol = 'https'
+const port = searchConfig.port
+const auth = `${searchConfig.username}:${encodeURIComponent(
+  searchConfig.password,
+)}`
+
+let connectionOptions
+if (baseConfig.nodeEnv === Environment.Dev) {
+  connectionOptions = { rejectUnauthorized: false } // Turn off certificate verification (rejectUnauthorized: false)
+} else {
+  connectionOptions = {
+    ca: fs.readFileSync('/etc/ssl/cert.pem'), // Path to CA certificate for Alpine Linux, which is the OS AskGov runs on
+  }
+}
+
+export const searchClient = new Client({
+  node: protocol + '://' + auth + '@' + searchHost + ':' + port,
+  ssl: connectionOptions,
+})
```

Comment:
So what about...
```suggestion
const ssl = baseConfig.nodeEnv === Environment.Dev
  // Turn off certificate verification (rejectUnauthorized: false) 
  // if we are running in a dev environment
  ? { rejectUnauthorized: false } 
  // Otherwise, set the path to CA certificate for Alpine Linux, 
  // the OS that AskGov runs on
  : { ca: fs.readFileSync('/etc/ssl/cert.pem') } 

export const searchClient = new Client({
  node: `${protocol}://${auth}@${searchHost}:${port}`,
  ssl,
})
```

Reply:
okay done, thanks!